components/io_expander/esp_io_expander_tca95xx_16bit/CMakeLists.txt

```
@@ -0,0 +1 @@
+idf_component_register(SRCS "esp_io_expander_tca95xx_16bit.c" INCLUDE_DIRS "include" REQUIRES "driver" "esp_io_expander")
```

```suggestion
idf_component_register(SRCS "esp_io_expander_tca95xx_16bit.c" INCLUDE_DIRS "include" REQUIRES "driver")
```

Ntipick: The dependency on 'esp_io_expander' is defined in idf_component.yml and it should not be redefined here